### PR TITLE
Update to standard-engine 3 and eslint 2

### PR DESCRIPTION
**Pull request by misteroneill**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,14 +4,16 @@
...
   "author": "Brightcove, Inc.",
   "bin": {
-    "vjsstandard": "./bin/cmd.js"
+    "vjsstandard": "./cli.js"
   },
   "bugs": {
     "url": "https://github.com/videojs/standard/issues"
   },
   "dependencies": {
+    "babel-eslint": "^6.0.0",
```
Comment (gkatsev) [inline package.json]: does babel-eslint 6.x require babel 6.x?

Comment (misteroneill) [inline package.json]: It uses babylon 6.x, but I don't think it's harmful - it's really only there to parse ES6 code and doesn't interact with the rest of our babel 5.x stuff.

**Comment by gkatsev** [comment]
LGTM

**Comment by incompl** [comment]
Lgtm2

**Pull request merged**